tests/pipelines/test_pipelines_document_question_answering.py

```diff
@@ -103,7 +103,7 @@ def run_pipeline_test(self, dqa_pipeline, examples):
     @require_detectron2
     @require_pytesseract
     def test_small_model_pt(self):
-        dqa_pipeline = pipeline("document-question-answering", model="hf-internal-testing/tiny-random-layoutlmv2")
```

Comment:
this checkpoint doesn't contain all weights, and random init. doesn't reproduce in each run.
(`LayoutLMv2PreTrainedModel._init_weights` is not written to take into account `LayoutLMv2VisualBackbone` added later)